include/envoy/upstream/load_balancer_type.h

```diff
@@ -17,6 +19,11 @@ namespace Upstream {
  */
 enum class LoadBalancerType { RoundRobin, LeastRequest, Random, RingHash, OriginalDst, Maglev };
 
+typedef std::pair<
+    std::set<std::string>,
+    absl::optional<envoy::api::v2::Cluster::LbSubsetConfig::LbSubsetSelectorFallbackPolicy>>
```

Comment:
Does this actually need to be absl::optional<>? The default value of this enum is NOT_DEFINED_PER_SELECTOR which if I read the code correctly is equivalent to the "empty" optional value.